mmv1/third_party/terraform/services/dataplex/resource_dataplex_glossary_test.go.tmpl

```diff
@@ -0,0 +1,82 @@
+package dataplex_test
+
+{{- if ne $.TargetVersionName "ga" }}
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+)
+
+func TestAccDataplexGlossary_update(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"project_name":  envvar.GetTestProjectFromEnv(),
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderBetaFactories(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataplexGlossary_update(context),
+			},
+			{
+				ResourceName:            "google_dataplex_glossary.glossary_test_id",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"glossary_id", "labels", "location", "terraform_labels"},
+			},
+			{
+				Config: testAccDataplexGlossary_full(context),
```

Comment:
This needs ConfigPlanChecks as described on https://googlecloudplatform.github.io/magic-modules/test/test/#add-an-update-test to ensure the resource isn't being deleted & recreated.

```suggestion
				Config: testAccDataplexGlossary_full(context),
				ConfigPlanChecks: resource.ConfigPlanChecks{
					PreApply: []plancheck.PlanCheck{
						plancheck.ExpectResourceAction("google_dataplex_glossary.glossary_test_id", plancheck.ResourceActionUpdate),
					},
				},
```